Makefile

```diff
@@ -92,19 +92,25 @@ appstore:
 	--exclude=.jscsrc \
 	--exclude=.jshintignore \
 	--exclude=js/views/templates \
+	--exclude=js/**.js.map \
```

Comment:
https://webpack.js.org/configuration/devtool/

> **source-map**
> A full SourceMap is emitted as a separate file. It adds a reference comment to the bundle so development tools know where to find it.
> You should configure your server to disallow access to the Source Map file for normal users!

So we just remove this from the package and users can upload the maps in case they need to debug something.